### DISCUSSION TITLE
VS Code Theme whitescreens IOS app

### DISCUSSION THREAD

**Discussion by newbery** [standardnotes/forum#2592] [General]
**Describe the bug**
The VS Code Theme appears to result in iphone white screen of death

**To Reproduce**
1. In desktop app, switch to VS Code Theme
2. In ios app, attempt to create a new note. A few seconds in (maybe the delay before the theme switch is synced), and then the screen goes completely white.
3. Go back to desktop app, and switch back to Default appearance. Wait a few seconds and then the display for the ios app becomes visible again.

**Expected behavior**
Obviously, changing theme in the desktop app should not kill the ios app.

**Desktop:**
 - OSX 12.6
 - Standard Notes 3.23.238
 
**Smartphone:**
 - iPhone SE, ios 15.6.1
 - Standard Notes 3.43.5

**Comment by effieeee** [comment]
Hi @newbery, can you send me a link to this particular theme so that I can test this out? Thank you!

**Comment by newbery** [comment]
Ahh. I had forgotten this was a third-party theme. I should probably repost this report over there?
https://github.com/hyphone/sn-theme-vscode

Done... https://github.com/hyphone/sn-theme-vscode/issues/6

**Comment by newbery** [comment]
Which just occurs to me... is there an easy way to export all third-party extensions in use? The list under "Account Settings >> General >> Advanced Options" doesn't really do a good job of telling me which of these are third-party extensions or give me any links to the extension sources. I'm guessing I can spelunk into the data itself but it was easier to do a google search to find this info.

**Comment by newbery** [comment]
Found it under ~/Library/Application Support/Standard Notes/Extensions/...

`{
    "identifier": "sn-vscode-theme",
    "name": "VS Code Theme",
    "content_type": "SN|Theme",
    "area": "themes",
    "version": "1.0.16",
    "download_url": "https://marcaux.de/sn/vscode-theme/sn-vscode-theme.zip",
    "url": "https://marcaux.de/sn/vscode-theme/dist/dist.css",
    "latest_url": "https://marcaux.de/sn/vscode-theme/ext.json",
    "thumbnail_url": "https://marcaux.de/sn/vscode-theme/screenshot.png",
    "dock_icon": {
      "type": "circle",
      "background_color": "#252526",
      "foreground_color": "#D4D4D4",
      "border_color": "#7ba78a"
    },
    "statusBar": "dark-content"
}
`

**Comment by newbery** [comment]
Hmm... I'm guessing that location for Extensions data is no longer current? It doesn't appear to be updated when I uninstall extensions.

**Comment by effieeee** [comment]
Thank you for providing the link and some additional info while you looked into things! When I tested this out on iPad though, I'm afraid I was still able to create new notes while the VSCode theme was set. The theme has also properly adapted to the other note types so far. Does the behavior still persist for you, and does it happen regardless of what note type you're using?
